DBZ-6632 Improve properties description on JDBC connector documentation (#4655)
debezium/debezium: documentation/modules/ROOT/pages/connectors/jdbc.adoc

<10> Enables the connector to evolve the destination database's schema.
For more information, see the xref:jdbc-property-schema-evolution[schema.evolution] configuration property.
<11> Specifies the timezone used when writing temporal field types.

> <12> List of topics to consume, separated by commas

Reviewer: ```suggestion
<12> List of topics to consume, separated by commas.
```

Many properties have default values.
Information about the properties is organized as follows:

* xref:jdbc-connector-properties-generic[]
* xref:jdbc-connector-properties-connection[]
* xref:jdbc-connector-properties-runtime[]
* xref:jdbc-connector-properties-extendable[]

> 

Reviewer: I missed this earlier, but these link lists that appear in the documentation for the other connectors are designed for downstream use (conditionalized as `product`]) , where the specified IDs match anchors applied to the downstream version of the docs. I forget now if and when we're planning to expose the JDBC connector downstream, but if we do, we'll have to make sure that we retain the upstream anchor IDs vs. using the `ModuleID` comment to modify the IDs for downstream. Otherwise the empty link text causes problems.
 Just a comment for now ... no action required.

Reviewer 2: We likely will at some future point, I can't say when @roldanbob, so if it makes less work for later to fix this to be compliant for what we may need, I'm all for it.

Author: No issue! I'll merge.

[[jdbc-connector-properties-generic]]
.Generic properties
[cols="30%a,25%a,45%a"]
|===
|Property |Default |Description

|[[jdbc-property-connection-name]]<<jdbc-property-connection-name, `+name+`>>
|No default
|Unique name for the connector. Attempting to register again with the same name will fail. This property is required by all Kafka Connect connectors.

|[[jdbc-property-connection-class]]<<jdbc-property-connection-class, `+connector.class+`>>
|No default

> |The name of the Java class for the connector. Always use a value of `io.debezium.connector.jdbc.JdbcSinkConnector` for the JDBC connector.

Reviewer: ```suggestion
|The name of the Java class for the connector. 
For the {prodname} JDBC connector, specify the value `io.debezium.connector.jdbc.JdbcSinkConnector`.
```

|[[jdbc-property-connection-task]]<<jdbc-property-connection-task, `+tasks.max+`>>
|1
|Maximum number of tasks to use for this connector.

|[[jdbc-property-connection-topics]]<<jdbc-property-connection-topics, `+topics+`>>
|No default

> |List of topics to consume, separated by commas.

Reviewer: ```suggestion
|List of topics to consume, separated by commas.
Do not use this property in combination with the xref:jdbc-property-connection-topics-regex[`topics.regex`] property.
```
